projects/commudle-admin/src/app/feature-modules/labs/components/labs/search-bar/search-bar.component.scss

```diff
@@ -0,0 +1,137 @@
+@import '/projects/commudle-admin/src/assets/styles/variables.scss';
+
+$primary-color: #536EFF;
+$secondary-color: #EEEEFF;
+
+$content-color: #2F2E41;
+
+$spacing: 16px;
```

Comment:
Let's rename this to $space-16

Reply:
also, let's define all these variables in variables.scss

Reply:
and get better names (as in if it is violet, then bg-violet, light-bg-violet or something on those lines

Reply:
Done